echoshader/echogram.py

```diff
@@ -146,7 +148,7 @@ def __init__(self, MVBS_ds):
 
         # http://holoviews.org/user_guide/Applying_Customizations.html
         self.gram_opts = holoviews.opts(
-            invert_yaxis=False, tools=["hover", "box_select"]
+            invert_yaxis=False, tools=["hover", "box_select", "lasso_select"]
```

Comment:
I tried the lasso but I could not understand how to use it.

Reply:
Nevermind, I tried lasso today and now it is working: yesterday I could make only a box with it?!

Reply:
Hi @valentina-s, now lasso option is just a option. We need to add corresponding function (plotting and data extracting) later.